payloads/external/skiboot/Makefile

```diff
@@ -0,0 +1,35 @@
+## SPDX-License-Identifier: GPL-2.0-only
+
+skiboot_dir=$(CURDIR)/talos-skiboot
+skiboot_git_repo=https://git.raptorcs.com/git/talos-skiboot
+skiboot_revision=9858186353f2203fe477f316964e03609d12fd1d
+dts=talos_ii_device-tree_stripped.dts
+skiboot=skiboot.lid
+uimage=skiboot.uimage
+config=config.its
+
+unexport $(COREBOOT_EXPORTS)
+
+.PHONY: all clean distclean
+
+all: $(CURDIR)/$(uimage)
+
+$(CURDIR)/$(uimage): $(skiboot_dir)/$(skiboot) $(CURDIR)/fdt.bin
+	mkimage -f $(config) $(uimage)
+
+$(CURDIR)/fdt.bin:
```

Comment:
This target should depend on `$(dts)`. Also `fdt.bin` is not removed by `make clean/distclean`.

Reply:
Fixed